base_tier_validation/models/tier_review.py

```diff
@@ -123,3 +124,20 @@ def _get_reviewers(self):
             if not reviewer_field or not reviewer_field._name == "res.users":
                 raise ValidationError(_("There are no res.users in the selected field"))
         return reviewer_field
+
+    def _get_reminder_notification_subtype(self):
+        return "base_tier_validation.mt_tier_validation_reminder"
+
+    def _notify_review_reminder_body(self):
+        delay = (fields.Datetime.now() - self.create_date).days
+        return _("A review has been requested %s days ago.") % (delay)
+
+    def _notify_review_reminder(self):
+        record = self.env[self.model].browse(self.res_id)
+        post = "message_post"
+        if hasattr(record, post):
+            getattr(record.sudo(), post)(
```

Comment:
why posting a msg rather than create an activity?

Reply:
Actually it can be done only if the model inheriting from `tier.validation` also inherits from `mail.activity.mixin` and not only `mail.thread` :grimacing:

Reply:
worth mentioning in a comment?